### PR TITLE
Response dir

### PR DESCRIPTION
**Pull request by david-yz-liu**
Moved all Haskell response modules into a subdirectory.

@Ian-Stewart-Binks, while I do generally prefer component-based organization, I think we should work to make the response files thin enough that they're basically just serving content computed elsewhere, and this "elsewhere" can be organized by component. I also suspect that for beginners, it's helpful to put all the files which use a particular library (like blaze-html or persistent) in one place. Please let me know if you have any thoughts on this.

### CODE REVIEW

File: hs/Response.hs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import Response.About
+import Response.Calendar
+import Response.Draw
+import Response.Graph
+import Response.Grid
+import Response.Image
+import Response.NotFound
+import Response.Post
+import Response.Privacy
+import Response.Search
```
Comment (Ian-Stewart-Binks) [inline hs/Response.hs]: Newline.

File: hs/Response/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Responses
+=========
+
+This directory contains all of the Haskell files responsible for creating server responses.
+These modules should all be exported by `hs/Response.hs`.
+
+All of the responses contain a value `???response` of type `ServerPart Response`,
```
Comment (Ian-Stewart-Binks) [inline hs/Response/README.md]: Perhaps `???Response`.

**Comment by Ian-Stewart-Binks** [comment]
@david-yz-liu I agree. It makes it much more digestible.

**Comment by david-yz-liu** [comment]
Thanks, @Ian-Stewart-Binks!

**Pull request merged**